src/Octoshift/Handlers/WaitForMigrationCommandHandler.cs

```diff
@@ -76,11 +76,19 @@ private async Task WaitForOrgMigration(string migrationId, string githubPat, Git
                 throw new OctoshiftCliException($"Migration {migrationId} failed for {sourceOrgUrl} -> {targetOrgName}. Failure reason: {failureReason}");
             }
 
-            _log.LogInformation($"Migration {migrationId} is {state}");
+            if (OrganizationMigrationStatus.IsRepoMigration(state))
+            {
+                var completedRepositoriesCount = (int)totalRepositoriesCount - (int)remainingRepositoriesCount;
```

Comment:
since both total repo count and remaining repo count can be null, there's a potential of an exception here. So either we need to guard against that or just don't use null values.

Reply:
I did write (and then delete) a unit test with 2 nulls and it didn't crash (I had the same thought)

Reply:
It is guaranteed that when the repo is in the `REPO_MIGRATION` state - as is tested by the `if` condition - the counts will be present.